keylime/migrations/versions/f838d3cdeead_add_last_successful_quote.py

```diff
@@ -0,0 +1,41 @@
+"""add last_successful_quote
+
+Revision ID: f838d3cdeead
+Revises: 8c0f8ded1f90
+Create Date: 2023-02-15 10:51:25.948918
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f838d3cdeead'
+down_revision = '8c0f8ded1f90'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()[f"upgrade_{engine_name}"]()
+
+
+def downgrade(engine_name):
+    globals()[f"downgrade_{engine_name}"]()
+
+
+def upgrade_registrar():
+    pass
+
+
+def downgrade_registrar():
+    pass
+
+
+def upgrade_cloud_verifier():
+    op.add_column("verifiermain", sa.Column("last_successful_quote", sa.Integer(), nullable=True))
```

Comment:
Maybe I am wrong, but verifier_db says that it should not be nullable, I wonder if we can drop this here and put some initial value (0)

Maybe even a similar change can be done for last_received_quote (or not, as at this point can be too late in the migration)

Reply:
So, same problem. I remember an issue with at least one of the backends (MySQL, if memory serves) that lead me, after a quick chat with @mpeters to adopt the more laissez faire policy of set `nullable` to `True` and not setting a value there. I am not going to strongly advocate for it, but if we want to change, we might have to add in a migration for the other column too.